docs/rocm.md

```diff
@@ -1,27 +1,27 @@
 # What is ROCm?
 
-ROCm is an open-source stack for GPU computation. ROCm is primarily Open-Source
-Software (OSS) that allows developers the freedom to customize and tailor their
-GPU software for their own needs while collaborating with a community of other
-developers, and helping each other find solutions in an agile, flexible, rapid,
-and secure manner.
+ROCm is an open-source stack, composed primarily of open-source software (OSS), designed for 
+graphics processing unit (GPU) computation. ROCm consists of a collection of drivers, development 
+tools, and APIs that enable GPU programming from low-level kernel to end-user applications. 
 
-ROCm is a collection of drivers, development tools and APIs enabling GPU
-programming from the low-level kernel to end-user applications.
+With ROCm, you can customize your GPU software to meet your specific needs. You can develop, 
+collaborate, test, and deploy your applications in a free, open-source, integrated, and secure software 
+ecosystem. ROCm is particularly well-suited to GPU-accelerated high-performance computing (HPC),
+artificial intelligence (AI), scientific computing, and computer aided design (CAD). 
 
-ROCm is powered by AMD’s Heterogeneous-computing Interface for Portability (HIP),
-an OSS C++ GPU programming environment and its corresponding runtime.
-HIP allows ROCm developers to create portable applications on different platforms
-by deploying code on a range of platforms, from dedicated gaming GPUs to exascale
-High Performance Computing (HPC) clusters.
+ROCm is powered by AMD’s 
+[Heterogeneous-computing Interface for Portability (HIP)](https://github.com/ROCm-Developer-Tools/HIP), 
+an OSS C++ GPU programming environment and its corresponding runtime. HIP allows ROCm
+developers to create portable applications on different platforms by deploying code on a range of 
+platforms, from dedicated gaming GPUs to exascale HPC clusters.
 
-ROCm supports programming models such as OpenMP and OpenCL,
-and includes all the necessary OSS compilers, debuggers, and libraries.
-ROCm is fully integrated into ML frameworks such as PyTorch and TensorFlow. ROCm can be
-deployed in many ways, including through the use of containers such as Docker,
-Spack, and your own build from source.
+ROCm supports programming models, such as OpenMP and OpenCL, and includes all necessary OSS 
+compilers, debuggers, and libraries. ROCm is fully integrated into machine learning (ML) frameworks, 
+such as PyTorch and TensorFlow. 
 
-ROCm’s goal is to allow our users to maximize their GPU hardware investment.
-ROCm is designed to help develop, test and deploy GPU accelerated HPC, AI,
-scientific computing, CAD, and other applications in a free, open-source,
-integrated, and secure software ecosystem.
+You can deploy ROCm using:
```

Comment:
We might want to just move this stuff to a new page. I have left a deploy page missing as I was waiting for Windows to be released before doing this.